core/src/i18n/en.yaml

```diff
@@ -592,7 +587,7 @@ components:
     value: Value
   label-selector-input:
     click-to-delete: Click to delete
-    enter-key-value: Enter {{type}} key=value
+    enter-key-value: Enter key-value labels
```

Comment:
Not sure if we use it anymore (might require some component purge to happen - i'll investigate it independently)

Reply:
I mean, we DO use it, but in a component that might be not necessary anymore :)

Reply:
Since it is available in the Dashboard I'd suggest to keep it :)